target/linux/ar71xx/base-files/etc/board.d/02_network

```diff
@@ -260,6 +260,9 @@ ar71xx_setup_interfaces()
 	tl-wa801nd-v3)
 		ucidef_set_interface_lan "eth1"
 		;;
+	tl-wa801nd-v4)
```

Comment:
Merge with v3?

Reply:
I will merge this, as mentioned in the comment above.